docker-compose.yaml

```diff
@@ -3,6 +3,25 @@ version: '3.8'
 name: manvsim
 
 services:
+
+  postgres:
+    image: postgres:latest
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_USER: admin
+      POSTGRES_PASSWORD: t3zgVzcWTd3WV8KTmnNH
```

Comment:
Is the database accessible from outside the machine/network?

Reply:
moved to config file